client-src/modules/ansi-html/index.js

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
Not sure if `modules` is the correct directory for this, I see that webpack builds some separate bundles based on subdirectories of `modules`. Let me know if there is a better suited location.